skimage/morphology/binary.py

```diff
@@ -144,3 +144,35 @@ def binary_closing(image, selem=None, out=None):
     dilated = binary_dilation(image, selem)
     out = binary_erosion(dilated, selem, out=out)
     return out
+
+@default_selem
+def binary_perimeter(image, selem=None):
+    """Return perimeter mask of objects in a binary image
+
+    Find the perimeters of objects in a provided binary image for a given
+    connectivity
+    Uses padding and binary erosion
```

Comment:
+1. In the Notes section, please explain the algorithm: the perimeter is the original shape minus its erosion.